Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1355,6 +1356,37 @@ static inline double normalizedFlexFraction(const GridTrack& track)
     return track.baseSize() / std::max<double>(1, flexFactor);
 }
 
+void IndefiniteSizeStrategy::accumulateFlexFractionMasonry(double& flexFraction, unsigned trackIndex, GridTrackSizingDirection outermostDirection, GridLayoutState& gridLayoutState) const
+{
+    auto& allTracks = m_algorithm.tracks(outermostDirection);
+    auto trackLength = allTracks.size();
```

Comment:
We don't seem to use `allTracks` so maybe:


```suggestion
    auto numTracks = m_algorithm.tracks(outermostDirection).size()
```

Reply:
We only use `allTracks` in line 1362, so we can shorten this code by removing it.